backend/src/@types/knex.d.ts

```diff
@@ -1,4 +1,4 @@
-import { Knex } from "knex";
+import { Knex as KnexOriginal } from "knex";
```

Comment:
question: if KnexOriginal is the same as Knex, what does this do?

Reply:
This was renamed because of this line

https://github.com/Infisical/infisical/pull/2019/files/5d59fe8810668afb205b3cff57a8396c4d902844#diff-ae28c47becba560585c4ced163583dbcbf8516e29b8d68a0282d1b870b5518b9R284